src/System.Net.ServicePoint/tests/TlsSystemDefault.cs

```diff
@@ -13,13 +11,22 @@ public class TlsSystemDefault
         [Fact]
         public void ServicePointManager_SecurityProtocolDefault_Ok()
         {
-            RemoteExecutor.Invoke(() => Assert.Equal(SecurityProtocolType.SystemDefault, ServicePointManager.SecurityProtocol)).Dispose();
+            Assert.Equal(SecurityProtocolType.SystemDefault, ServicePointManager.SecurityProtocol);
         }
 
         [Fact]
         public void ServicePointManager_CheckAllowedProtocols_SystemDefault_Allowed()
         {
-            RemoteExecutor.Invoke(() => ServicePointManager.SecurityProtocol = SecurityProtocolType.SystemDefault).Dispose();
+            SecurityProtocolType orig = ServicePointManager.SecurityProtocol;
```

Comment:
It doesn't need to be remote? This won't be affected by concurrent tests?

Reply:
I disabled concurrency for the project:
https://github.com/dotnet/corefx/pull/41741/files/6fb4dc9e2df5ee046f1a72a2233a21512fd85ca1#diff-1fb5f1c19bceccae0e565700861e62b6R8